cnew.py

```diff
@@ -51,7 +51,10 @@ async def producer_handler(cphrsuit, websocket, username, chatroom, servaddr):
         footelem = HTML("<b>[" + chatroom + "]</b>" + " " + username.strip() + " - Sanctuary ZERO v04092020 running on '" + servaddr + "' - Hit Ctrl+C to EXIT")
         while True:
             with patch_stdout():
-                mesgtext = await sess.prompt_async(lambda:"[" + obtntime() + "] " + formusnm(str(username)) + " > ", bottom_toolbar=footelem, validator=emtyfind(), refresh_interval=0.5)
+                mesgtext = await sess.prompt_async(lambda:"[" + obtntime() + "] " + formusnm(str(username)) + " > ", bottom_toolbar=footelem,
+                                                                                                                     validator=emtyfind(),
+                                                                                                                     refresh_interval=0.5,
+                                                                                                                     prompt_continuation=lambda width, line_number, is_soft_wrap: ' ' * width)
```

Comment:
Yes, the latest commit on `master` does not seem to have the `prompt_continuation`. Thanks for adding it back.